src/main/java/org/example/pageobject/pages/listedproducts/ListedByPriceRangePage.java

```diff
@@ -0,0 +1,42 @@
+package org.example.pageobject.pages.listedproducts;
+
+import org.example.pageobject.BasePage;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import java.util.List;
+
+public class ListedByPriceRangePage extends BasePage {
+    @FindBy(xpath = "//*[@id=\"s-refinements\"]/div[12]/ul/li")
+            public List<WebElement> priceRangeCount;
```

Comment:
It is better to make these variables private. And format your code. Use CTRL + ALT + L

Reply:
Done!